torchmdnet/data.py

```diff
@@ -84,7 +85,7 @@ def std(self):
 
     def _get_dataloader(self, dataset, stage, store_dataloader=True):
         store_dataloader = (
-            store_dataloader and not self.trainer.reload_dataloaders_every_epoch
+            store_dataloader and not self.trainer.reload_dataloaders_every_n_epochs
```

Comment:
I would make this

```python
store_dataloader and self.trainer.reload_dataloaders_every_n_epochs == 0
```

to clarify that reload_dataloaders_every_n_epochs is an int, not a bool.